pkgs/development/python-modules/jinja2/default.nix

```
@@ -10,12 +10,12 @@
 
 buildPythonPackage rec {
   pname = "Jinja2";
-  version = "3.0.3";
+  version = "3.1.1";
   disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.7";
```
https://github.com/pallets/jinja/blob/3.1.1/setup.cfg#L36=